website/content/docs/connect/ca/vault.mdx

```diff
@@ -7,40 +7,29 @@ description: >-
+This page describes how configure the Vault CA provider.
 
-This page documents the specifics of the Vault CA provider.
-Please read the [certificate management overview](/docs/connect/ca)
-page first to understand how Consul manages certificates with configurable
-CA providers.
-
--> **Tip:** Complete the [tutorial](https://learn.hashicorp.com/tutorials/consul/vault-pki-consul-connect-ca?in=consul/vault-secure) to learn how to configure Vault as the Consul Connect service mesh Certification Authority.
+> **Tutorial:** Complete the [Vault as Consul Service Mesh Certification Authority](/consul/tutorials/vault-secure/vault-pki-consul-connect-ca) tutorial for hands-on guidance on how to configure Vault as the Consul service mesh certification authority.
 
 ## Requirements
 
-Prior to using Vault as a CA provider for Consul, the following requirements
-must be met:
-
-- **Vault 0.10.3 or later.** Consul uses URI SANs in the PKI engine which
-  were introduced in Vault 0.10.3. Prior versions of Vault are not
-  compatible with Connect.
+- Refer to [Service Mesh Certificate Authority Overview](/docs/connect/ca) for important background information about how Consul manages certificates with configurable CA providers. 
 
-~> **Note:** Do not use Vault v1.11.0+ as Consul's Connect CA provider — the intermediate CA will become unable to issue the leaf nodes required by service mesh, and by Consul client agents if using auto-encrypt or auto-config and using TLS for agent communication. If you are already using Vault 1.11+ as a Connect CA, refer to this [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/11308460105491) for more information about the underlying cause and recommended workaround.
+- Use Vault 0.10.3 to latest 1.10.x.
 
-## Configuration
+~> **Note:** Do not use Vault v1.11.0+ as Consul's Connect CA provider — the intermediate CA becomes unable to issue the leaf nodes required by service mesh, and by Consul client agents if using auto-encrypt or auto-config and using TLS for agent communication. If you are already using Vault 1.11+ as a Connect CA, refer to this [Knowledge Base article](https://support.hashicorp.com/hc/en-us/articles/11308460105491) for more information about the underlying cause and recommended workaround.
 
-The Vault CA is enabled by setting the CA provider to `"vault"` and
-setting the required configuration values.
+## Enable Vault as the CA
 
-The configuration may either be provided in the agent's configuration file using
-the [`ca_provider`] and [`ca_config`] options, or configured using the
-[`/connect/ca/configuration`] API endpoint.
+You can enable Vault as the CA by configuring Consul to use `"vault"` as the CA provider
+and including the required provider configuration options.
+You can provide the CA configuration in the agent configuration file of server agents,
```

Comment:
I hesitated to write it this way to begin with because their are multiple types of files that can contain configuration ... "configuration entries" are different from "agent configuration".

That said, perhaps it's better overall to make this suggested change, even though there may be a possibility of misinterpretation.